File: examples/psamples.py
#!/usr/bin/env python
#
# file: psamples.py		G. Moody	11 March 2006 	
#                               last modified:  14 January 2013
#
# Python translation of psamples.c from the WFDB Programmer's Guide 	

import sys
import wfdb

def main(argv):
    siarray = wfdb.WFDB_SiginfoArray(2)
    if wfdb.isigopen("100s", siarray, 2) < 2: sys.exit(1)
    v = wfdb.intArray(2)
    for i in range(0,10):
        if wfdb.getvec(v) < 0: sys.exit(2)
        print "\t%d\t%d" % (v[0], v[1])

if __name__ == "__main__":
    main(sys.argv[1:])

File: examples/psamples.cs
// file: psamples.cs		G. Moody	5 April 2006
//
// C# translation of psamples.c from the WFDB Programmer's Guide 	

using System;
using Wfdb;

public class psamples {
    static void Main(string[] argv) {
        WFDB_SiginfoArray siarray = new WFDB_SiginfoArray(2);
        if (wfdb.isigopen("100s", siarray.cast(), 2) < 2)
            Environment.Exit(1);
        WFDB_SampleArray v = new WFDB_SampleArray(2);
        for (int i = 0; i < 10; i++) {
            if (wfdb.getvec(v.cast()) < 0)
                break;
	    Console.WriteLine("\t" + v.getitem(0) + "\t" + v.getitem(1));
        }
    }
}
